lib/1_is_greater_than_five.rb

```diff
@@ -5,5 +5,9 @@
 # Run `bin/check` from the command line to execute the automated tests.
 
 def is_greater_than_five(num)
-  # Write your code here.
+    if num > 5
```

Comment:
This is great! Is it possible to condense this method to one line? (Think about how the expression `num > 5` evaluates i.e. what it returns)